File: algorithm-one/string-manipulation-1/src/is-vowel.js
/* exported isVowel */
function isVowel(character) {
  const vowels = {
    'a': true,
    'A': true,
    'e': true,
    'E': true,
    'i': true,
    'I': true,
    'o': true,
    'O': true,
    'u': true,
    'U': true
  }
  if (vowels[character]) return true;
  return false;
}

File: algorithm-one/string-manipulation-1/python/is-vowel.py
vowel = {
  'a': True,
  'e': True,
  'i': True,
  'o': True,
  'u': True,
  'A': True,
  'E': True,
  'I': True,
  'O': True,
  'U': True
}

def isVowel(char):
  if (vowel.get(char)):
    return True
  else:
    return False


print(isVowel('L'))
print(isVowel('f'))
print(isVowel('Z'))
print(isVowel('a'))
print(isVowel('E'))
print(isVowel('I'))
print(isVowel('o'))
print(isVowel('u'))
